Bashbashbash patch 1

bashbashbash opened the pull request:
Updated for issue #2

kastolars committed:
Merge pull request #6 from SIGBlockchain/dev

modified script

kastolars committed:
Merge pull request #7 from SIGBlockchain/dev

Revert "Merge branch 'master' into dev"

bashbashbash committed:
Update README.md

Updated for issue #2

kastolars reviewed (changes requested):
I have added a checklist to the issue to show what is only needed.

bashbashbash committed:
Update readme for issue #2

Update readme for issue #2

bashbashbash commented:
updated

kastolars commented on README.md:
```diff
@@ -1 +1,57 @@
+Download this branch into your local environent. 
+Open a console or terminal window and navigate to the the aurum_client directory
+Type the following:
+sudo chmod +x setup.sh
```
I'm not sure these particular instructions will work for all OS's, perhaps specify that?

kastolars commented on README.md:
```diff
@@ -1 +1,57 @@
+Run ./client -h or ./client --help to see all commands
+Check out the wiki on the github page for more info: https://github.com/SIGBlockchain/aurum_client/wiki
+
+A file called aurum_wallet.json was created. Copy this file and store it somewhere secure. __It contains your private key. If you lose you private key, you will lose access to your wallet and it is not recoverable.__
```
Urge the user to save a copy of `aurum_wallet.json` to another secure location just in case.

kastolars commented on README.md:
```diff
@@ -1 +1,57 @@
+[Linux Binary](https://github.com/SIGBlockchain/aurum_client/tree/linux_release)
+[Mac OS X Binary](https://github.com/SIGBlockchain/aurum_client/tree/mac_release)
+[Windows Binary](https://github.com/SIGBlockchain/aurum_client/tree/windows_release)
+Download this branch into your local environent. 
```
Can we get a little more specific with what exactly to download? Perhaps just mention the zip folder and how to get the contents of it.

bashbashbash committed:
Updated README.md 

Added further clarification to installation instructions.

bashbashbash committed:
Update README.md

Final update based on commets

kastolars commented on README.md:
```diff
@@ -1 +1,58 @@
+Open a console or terminal window and navigate to the the aurum_client directory
+Type the following (if on a Linux / Mac OS X system - for Windows, please change the listed files to be executable and run):
+sudo chmod +x setup.sh
+sudo ./setup.sh
```
I don't think sudo runs on Windows

kastolars commented on README.md:
```diff
@@ -1 +1,58 @@
+[Mac OS X Binary](https://github.com/SIGBlockchain/aurum_client/tree/mac_release)
+[Windows Binary](https://github.com/SIGBlockchain/aurum_client/tree/windows_release)
+Navigate to the bright green button on the right hand side that says clone or download. Click download zip.
+Find the zip file you downloaded and unzip to your desired directoy.
```
For these binaries, can you simply add the link for the zip file?

kastolars commented on README.md:
```diff
@@ -1 +1,58 @@
+  -version
+    	client version
+
+string means put the address of the person you want to interact with.
```
What about for the `send` flag?

bashbashbash committed:
Updated the formatting and minor changes

Updated the formatting and minor changes - made sure links connect to zips and updated send explanation

bashbashbash committed:
Update - Header/Title

Update - Header/Title
